examples/common/pigweed/rpc_services/internal/StatusUtils.h

```diff
@@ -20,12 +20,13 @@
 #pragma once
 
 #include "app/util/attribute-storage.h"
+#include "protocols/interaction_model/StatusCode.h"
 #include "pw_status/status.h"
 
 #define RETURN_STATUS_IF_NOT_OK(expr)                                                                                              \
     do                                                                                                                             \
     {                                                                                                                              \
-        pw::Status __status = chip::rpc::EmberStatusToPwStatus(expr);                                                              \
+        pw::Status __status = chip::rpc::EmberOrChipStatusToPwStatus(expr);                                                        \
```

Comment:
instead of the Or, lets drop the `EmberStatus` prefix. We seem to just want an overload that can cast the `expr` into a pw::Status.

Reply:
Will do